.github/workflows/pull_push_request.yml

```diff
@@ -0,0 +1,43 @@
+# This is a CI workflow file created to execute on push and pull requests
+
+name: pull_push_workflow
+on: [pull_request, push]
+jobs:
+
+  kotlin-compiler:
+    runs-on: ubuntu-latest
```

Comment:
Here, test using the ubuntu LTS version 

like `runs-on: ubuntu-22.04`

Reply:
Done